etc/kayobe/stackhpc.yml

```diff
@@ -130,6 +130,23 @@ stackhpc_kolla_source_version: "stackhpc/{{ openstack_release }}"
 stackhpc_kolla_ansible_source_url: "https://github.com/stackhpc/kolla-ansible"
 stackhpc_kolla_ansible_source_version: "stackhpc/{{ openstack_release }}"
 
+###############################################################################
+# Overcloud host image source
+
+stackhpc_download_overcloud_images: false
+
+# Ubuntu overcloud host image source
+stackhpc_ubuntu_overcloud_host_image_url: "{{ stackhpc_release_pulp_content_url }}/kayobe-images/{{ openstack_release }}/ubuntu/focal/{{ stackhpc_ubuntu_overcloud_host_image_version }}/overcloud-ubuntu-focal.qcow2"
+stackhpc_ubuntu_overcloud_host_image_version: "latest"
```

Comment:
These should be pinned to a specific version.

Reply:
I've pinned them for now but should probably run the workflow again then test the new images and pin to them

Reply:
The parts of the config aimed at pulling overcloud host images have been removed while we wait for better support upstream.